src/emailLogin/removeGithubAccess.js

```diff
@@ -0,0 +1,31 @@
+const { Octokit } = require('octokit');
+
+// name of repo
+const REPO = process.argv[2];
+
+const { PERSONAL_ACCESS_TOKEN, GITHUB_ORG_NAME } = process.env;
+
+const octokit = new Octokit({
+  auth: PERSONAL_ACCESS_TOKEN,
+});
+
+const removeGithubAccess = async (site) => {
+  try {
+    await octokit.request(`PUT /orgs/${GITHUB_ORG_NAME}/teams/${site}/repos/${GITHUB_ORG_NAME}/${site}`, {
+      org: GITHUB_ORG_NAME,
+      team_slug: site,
+      owner: GITHUB_ORG_NAME,
+      repo: site,
+      permission: 'pull',
```

Comment:
clarifying question - this does set the permission to `pull` but do the original permissions still remain for the already added members of the site?

Reply:
This sets the access permissions of the team to `read`, and the members have access to the site only under this team, so their permissions will also be affected

Reply:
Hmm, I am confused by this. Shouldnt we remove the team entirely? ie, the only team with any access to the repo should be core?

Reply:
I think that's also possible! I opted to change their access to readonly so that we can still check on Github who the members of the team originally were - do you think we should remove the team entirely?

Reply:
I have a preference for removing as it is a cleaner solution! Also am worried about the edge case of: 
1. Agency has user X and user Y in their team. 
2. User X leaves the agency.
3. Agency requests to privatise their repos, repo is becomes private.
4. User X's github account still has read access to the repo. 
In essence, I think source of truth should always come from our db.
I think for logs, we can dump it to a file? and just put somewhere in our isomer folder if we need to parse it for audit, wdyt?

Reply:
Ah yup didn't think about private repos, good catch! Have changed it to remove access entirely in https://github.com/isomerpages/isomer-conversion-scripts/pull/56/commits/eff5b6840753c7335c6f359f86ba9ab098ee9d32